### ISSUE TITLE
False positive for consecutive ruby scripts sniff

### ISSUE THREAD

**Issue by oliverklee** [sds/haml-lint#36]
The following code creates warnings in haml-lint:

```
- if controller_name != 'sessions'
  = link_to 'Sign in',
    new_session_path(resource_name)
  %br

- if controller_name != 'registrations'
  = link_to 'Sign up',
    new_registration_path(resource_name)
  %br

- if controller_name != 'passwords' && controller_name != 'registrations'
  = link_to 'Forgot your password?',
    new_password_path(resource_name)
  %br

- if controller_name != 'confirmations'
  = link_to 'Didn\'t receive confirmation instructions?',
    new_confirmation_path(resource_name)
  %br

- if controller_name != 'unlocks'
  = link_to 'Didn\'t receive unlock instructions?',
    new_unlock_path(resource_name)
  %br
```

This is the warning:

```
app/views/devise/shared/_links.html.haml:1 [W] ConsecutiveSilentScripts: 5 consecutive Ruby scripts can be merged into a single `:ruby` filter
```

However, the template isn't pure Ruby - it contains the %br tags. haml-lint should recognize that this.
